inc/command/class-command.php

```diff
@@ -219,4 +225,21 @@ protected function get_config( $module = 'dev-tools' ) : array {
 		return (array) ( $composer_json['extra']['altis']['modules'][ $module ] ?? [] );
 	}
 
+	/**
+	 * Check if a given path is valid for PHPUnit.
+	 *
+	 * @param string $path The filepath to check.
+	 * @return boolean
+	 */
+	protected function is_valid_test_path( string $path ) : bool {
+		$full_path = $this->get_root_dir() . DIRECTORY_SEPARATOR . $path;
+		if ( strpos( $path, '*' ) !== false ) {
+			return ! empty( glob( $full_path ) );
+		}
+		if ( is_file( $full_path ) ) {
+			return pathinfo( $full_path, PATHINFO_EXTENSION ) === 'php';
```

Comment:
Is it necessary to check the extension here? The other tests don't validate this strictly, and since it's possible to include PHP files with different extensions, it seems it should also be possible to test them with command-line flags.

Reply:
Yeah the whole thing is a little strange because we're not really handling parsed options. It would require passing everything in the generated config file as command line arguments instead and checking for whether they have been set so they can be overridden. You still need a `phpunit.xml` if you want to test multiple directories though so I'm a little hamstrung with this current implementation.

You're right about the file extension though, I was maybe overthinking if the command was something like `composer dev-tools phpunit -- --coverage clover.xml` or something and there was an existing `clover.xml` from another test run.

It seems rare that other extensions are used so I could just extend it to include `php` and `inc`. I think it's a pretty rare and unusual use-case for any other extension to be used.